src/utils/pinned-entity-to-gltf.js

```diff
@@ -32,7 +32,10 @@ export default function pinnedEntityToGltf(el) {
 
     gltfComponents.media = { src: mediaSrc, contentSubtype: mediaContentSubtype, id: networkId };
 
-    if (components["media-pager"]) {
+    if (components["media-pdf"]) {
+      gltfComponents.media.pageIndex = components["media-pdf"].data.index;
+    } else if (components["media-pager"]) {
+      // Legacy
```

Comment:
Since this is only for serializing, do we need to support legacy?

Reply:
oh good catch, i mis-interpreted it as I was going through the code that this was a deserializer